client/src/views/Simulation/components/ModelPanel.vue

```diff
@@ -39,28 +45,53 @@
     @Prop({ default: false }) expanded: boolean;
     @Prop({ default: null }) model: Model.Model;
 
-    @Getter getSimParameters;
-    @Getter getSimVariables;
+    @Getter getSimModel;
+    @Getter getSelectedModelGraphType;
     @Action toggleParameter;
     @Action toggleVariable;
 
     settingsOpen: boolean = false;
 
     onNodeClick (selected: Graph.GraphNodeInterface): void {
```

Comment:
I think this method should be probably called `onNodeDblClick` since it is not just a simple click, it gets triggered on double click.